_posts/2020-02-21-centos7-python2-end-of-life.md

```diff
@@ -0,0 +1,108 @@
+---
+title:  "CentOS 7, Ansible and the end of Python 2"
+author: Dr John A Stevenson
+categories:
+  - devops
+tags:
+  - Python
+  - CentOS
+  - Ansible
+  - continuous-integration
+---
+
+The [sunsetting of Python 2](https://www.python.org/doc/sunset-python-2/) on 1 January 2020 has been causing problems our CentOS 7 server administration.
+We use [Ansible](https://www.ansible.com/) for automatic configuration and
+a few of our long-used jobs recently began to fail as a result Python 2 / Python 3 mismatches.
+Officially, CentOS 7 is [supported until
+June 2024](https://wiki.centos.org/About/Product), but I think the end of
+Python 2 will hasten its demise.
+
+The notes below explain how we solved our issues with CentOS 7 and Ansible.
+Hopefully they will be helpful to others.
+
+
+## Background
```

Comment:
Might be a template issue, but on my browser this renders as "BackgroundPermalink" (with the link under the Permalink text)

Reply:
And all the other headings and subheadings